test/JDBC/jdbc_schedule

```diff
@@ -33,6 +33,9 @@ ignore#!#BABEL-3117-vu-prepare
 ignore#!#BABEL-3117-vu-verify
 ignore#!#BABEL-3655-vu-prepare
 ignore#!#BABEL-3655-vu-verify
+ignore#!#GRANT_SCHEMA-vu-prepare
```

Comment:
What would cause failure in normal JDBC run?

Reply:
`GRANT_SCHEMA-vu-verify` would fail due to difference in few cases while handling upgrade since. Those scenarios are mentioned in the test plan added in the description.